pull_request_template.md

```diff
@@ -1,25 +1,36 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  
--  
+-  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
+-  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
+-  [ ] This pull request promotes database changes.
+
+If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.
 
 ## Issues Addressed:
 <!-- If your fix has a relating issue, link it below -->
 - Closes 
 
 ## SOURCE:
 <!-- If you can, include a source that can strengthen your claim -->
+- [ ] The changes promoted by this pull request have been validated through live research (checked on live servers)
+- [ ] The changes promoted by this pull request have been validated through sniffs (remember to share them with the opensource community!)
+- [ ] The changes promoted by this pull request come partially or entirelly from another project (cherry-pick). **Cherry-picks must be commited using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**
```

Comment:
Does not include any sort of 'hearsay' type sourcing, such as Wowhead, forum posts, etc. While obviously not optimal, these are *often* the source used for various changes and should be included. I would also personally appreciate asking for the commit hash or PR link as well for cherry-picks as it makes things easier if the change has to be evaluated down the road.

Grammar-wise,
L17 `opensource` -> `open-source`/`open source`
L18 `entirelly` -> `entirely`

Reply:
> I'm not very fond of hearsay as a source but I guess we can find a way to word it in

If only we had the luxury to exclude it. 😛